src/lib/resource/sections/applicant-future-work.js

```diff
@@ -43,7 +42,7 @@ module.exports = {
             errorMessage: {
                 required: {
                     'q-applicant-future-work':
-                        "Select yes if the victim's ability to work in future has been affected by their injuries"
+                        'Select yes if their ability to work has been affected by their injuries'
```

Comment:
@armoj sorry, I missed `in future` from the error message. Can you add it back in please? Thanks